src/components/NoteDetails/NoteDetailsTopBar.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import {
+  IconButton,
+  makeStyles,
+  Menu,
+  MenuItem,
+  Typography,
+} from '@material-ui/core';
+import { TopBar } from 'components/TopBar/TopBar';
+import { TopBarLeftButtons } from 'components/TopBar/TopBarLeftButtons';
+import MoreVertIcon from '@material-ui/icons/MoreVert';
+import { useDispatch, useSelector } from 'react-redux';
+import { useRef, useState } from 'react';
+import { TopBarPageTitle } from 'components/TopBar/TopBarPageTitle';
+import { changeParentButtonClickedAction } from 'components/BottomBar/BottomBarActions';
+import { colors } from 'colors';
+import { BackButton } from 'components/BackButton/BackButton';
+
+const useStyles = makeStyles(() => ({
+  moreIcon: {
+    position: 'absolute',
```

Comment:
Did you try using `TopBarRightButtons` component? I think it already has styles to position the buttons correctly on the right side of the topbar.